Classes/Base.php

```diff
@@ -278,4 +278,30 @@ public static function logout()
 			$session->set( $key, null );
 		}
 	}
+
+	/**
+	 * Doing a complete ACPu wipe with the red system cache.
+	 *
+	 * @hook clearCachePostProc
+	 *
+	 * @param array $cacheType The caching type.
+	 *
+	 * @return void
+	 */
+	public static function clearAPCuCache( array $cacheType )
```

Comment:
I would use a shorter name like `clearCache` that also works if more cache clearing has to be done in the future.

Reply:
Heh, I clicked accidentally on resolve . . .

Oh, and done.